lighthouse-testnet.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.0"
+
+####
+# https://github.com/sigp/lighthouse/tree/master/scripts/local_testnet
+
+services:
+  beacon_node:
+    image: "${DOCKER_LIGHTHOUSE_IMAGE}"
+    container_name: "eth2_testnet_beacon_lighthouse"
+    volumes:
+      - ./testnet/lighthouse/data:/root/testnet
+      - ./testnet/lighthouse/data/beacon:/root/beacon
+      - ./testnet/lighthouse:/root/scripts
+    ports:
+      - 5052:5052/tcp
+      - 9000:9000/udp
+      - 9000:9000/tcp
```

Comment:
remove 9000 ports. Although since newtrok is host, you don't need to expose any port anyways